st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -48,6 +52,17 @@ class ActionAliasController(resource.ContentPackResourceController):
         'sort': ['pack', 'name']
     }
 
+    _custom_actions = {
```

Comment:
Actually, it's a CLI issue - CLI doesn't correctly send query param to the method...


```python
match() takes exactly 2 arguments (1 given)
```

I will make `urlencode` and requests changes mentioned below, I believe this should fix it.